modules/ROOT/pages/writing.adoc

```diff
@@ -611,21 +611,28 @@ For `Overwrite` mode you *must have unique constraints on the keys*.
 
 === Schema optimization operations
 
+
 The Spark Connector supports schema optimization operations via:
 
-* index.
-* constraints.
+* index (deprecated);
+* constraints;
+* property type enforcement;
 * set of schema queries.
 
 To speed up the import itself that is executed *before* the import process starts.
 
+
+[NOTE]
+since the version **5.3.0** option **schema.optimization.type** is deprecated in favor of **`schema.optimization.node.\*`**, **`schema.optimization.relationship.*`** and **schema.type.constraint**.
+
+==== Deprecated approach
```

Comment:
```suggestion
==== With indexes (Deprecated)
```

Reply:
Mmm I disagree about this. It's not only related to indexes, I would rather say:
```suggestion
==== `schema.optimization.type` (Deprecated)
```
I think is a best fit

Reply:
No because it describes both indexes and constraints.
`schema.optimization.type` allows to create indexes and constraints deprecating this approach we do not allows to create indexes anymore but only constraints as is the goal of the new feature.

Reply:
Ok, I understand now. Sounds good then.